CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+# Contributing to ISIS3
+# Under Construction
+
+#### Table of Contents
+[I have a question or a problem!](#i-have-a-question-or-a-problem)
+
+[Redmine Issues](#redmine-issues)
+
+[How can I contribute?](#how-can-i-contribute)
+  * [Report an issue](#redmine-issues)
+  * [Suggesting enhancements](#redmine-issues)
+  * [Working on an existing issue or enhancement](#working-on-an-existing-issue-or-enhancement)
+  * [Working on a new issue or enhancement](#working-on-a-new-issue)
+
+[Code Base Contribution Guidelines](#code-base-contribution-guidelines)
+
+[What can I expect from the ISIS3 development team](#What-can-I-expect-from-the-ISIS3-development-team)
+  * [Response time](#Response-time)
+  * [Feedback](#feedback)
+
+[Development Process Checklists](https://github.com/USGS-Astrogeology/ISIS3/wiki/Developer-Checklists)
+
+### References
+  * [ISIS3 API Reference](https://isis.astrogeology.usgs.gov/Object/Developer/index.html)
+  * [Tutorials](https://isis.astrogeology.usgs.gov/fixit/projects/isis/wiki/ISIS_Online_Workshops)
+  * [Application Documentation](https://isis.astrogeology.usgs.gov/Application/index.html)
+  * [Coding Standards and Style Guide](https://isis.astrogeology.usgs.gov/documents/CodingStandards/CodingStandards.html)
+  * [Documentation](https://isis.astrogeology.usgs.gov/documents/CodingStandards/CodingStandards.html#documentation)
+  * [Redmine Issue Lifecycle](https://isis.astrogeology.usgs.gov/fixit/projects/isis/wiki/Life_Cycle_of_an_ISIS_Issue)
+  * ADD MORE THINGS!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+  
+## I have a question or a problem!
+If you have a question about or a problem with ISIS3, please see the [Redmine Issues](#redmine-issues) section.
+
+If you have a question or a problem with contributing to our software please contact ## REGENT EMAIL HERE ##.
+
+## How can I contribute?
+
+### Redmine Issues
+If you have a question about, a problem with, or a suggestion for ISIS3, please read our [Guidelines for reporting Redmine issues](https://isis.astrogeology.usgs.gov/fixit/projects/isis/wiki/Guidelines_for_Reporting_Issues). Once you have read that please [submit](https://isis.astrogeology.usgs.gov/fixit/projects/isis/issues/new) a Redmine issue.
+
+### Working on an existing issue or enhancement
+1. Assign the redmine issue to yourself. If you don't have the permission to do this, please make a note on the issue that you would like to work on it. Please contact ## REGENT EMAIL HERE ## for permission to work on an issue.
```

Comment:
I don't know which is correct, but it might be good to be consistent with Redmine/redmine capitalization